content/en/docs/guides/input_and_access.md

```diff
@@ -116,7 +113,7 @@ Defines a resource from content of a directory in the file system. It is packed
 
 #### docker
 
-Takes an image from the local docker registry and adds it as a resource. Requires a running docker daemon.
+Takes an image from a local docker registry and adds it as a resource. Requires a running docker daemon.
```

Comment:
There is always only one local docker registry. Therefore "the" is the better fit here.